### DISCUSSION TITLE
Take folder view and external libraries to the next level

### DISCUSSION THREAD

**Discussion by raulomator** [immich-app/immich#12710] [Feature Request]
Hi.

First, sorry for maybe creating the same idea twice. I used search, but didn't found the same request.
Second, GREAT SOFTWARE! PLEASE NEVER GIVE UP! :)

My Family loves Immich and we used it a lot, before we got new phones. I will try to explain why.
We switched from saving "a lot of albums" on phone and uploading them to Immich, then creating shared albums to using backups as mounted paths in external libraries. Configuring a lot of mounted paths in a docker-compose configfile is not great, but ok, since the paths do not change every week. This strategy will reduce our phone backup time spend in the future tremendously.

Our libraries are created, the paths are mounted and we see our pictures in our timeline, everything great so far.

--- Feature requests
1. Powerful folder view
I would like to be able to mark pictures in the folder view just like in the main timeline to give them a tag or directly sent them to an album. In other words, please change the folder view, so it has the same feature set as the timeline.

2. Auto create album
It would be great to tell Immich to automatically create an album out of an mounted imported path (maybe a slider in "Edit import path").
At the moment, external libraries are not really helpful for us, cause we got thousands of photos from a lot of mounted folders in there. It is almost impossible and at least really annoying to sort them out manually, pick the right photos and send them to an album.

3. Better search filter
Create a search filter that shows only picked mounted paths.

I think many people would love to see these improvements.
As soon as these changes are implemented and released, I will get a server license.

Greetings
Raul

**Comment by OlJohnny** [comment]
+1 for making the folder view as powerful as the timeline view

**Comment by Sohardh** [comment]
Same much needed!

**Comment by bo0tzz** [comment]
This is a dupe of several existing requests

**Reply by raulomator** [reply]
Thanks for clarifying. Please link the appropriate feature requests here, so everybody can move there and give them a boost.

**Reply by nielstron** [reply]
selecting multiple images in folder view is appearently a duplicate of https://github.com/immich-app/immich/discussions/9159 (because, if I understand correctly, folder view is a "filtered view"?)
